roles/gateway_import_database/README.md

```diff
@@ -5,6 +5,9 @@ This role will import Gateway and OTK database dump from ansible controller to d
 
 Requirements
 ------------
+* gateway database has to be created before running import script. 
```

Comment:
There is a task to fix capitalization/syntax/grammar etc in all docs, but might wanna patch this one now.